packages/dds/tree/src/shared-tree/independentView.ts

```diff
@@ -117,21 +110,55 @@ export function independentInitializedView<const TSchema extends ImplicitFieldSc
 
 	const fieldCursors = fieldBatchCodec.decode(content.tree as JsonCompatibleReadOnly, context);
 	assert(fieldCursors.length === 1, 0xa5b /* must have exactly 1 field in batch */);
-	// Checked above.
-	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-	initializeForest(forest, fieldCursors[0]!, revisionTagCodec, idCompressor, false);
+
+	const out: TreeViewAlpha<TSchema> = independentInitializedViewInternal<TSchema>(
```

Comment:
Is it necessary to factor this out into a variable? I recall there are weird cases when it matters. Is this one of them?

Reply:
No. Fixed.